docker/manifest.yml

```diff
@@ -75,23 +75,24 @@ releases:
       <<: *cuda11_3_1
     cuda_prefix_url: "ubuntu2004"
 
-  centos8:
-    <<: *base_spec
-    templates_dir: templates/rhel
-    base_image: centos:8
-    add_to_tags: "centos8"
-    cuda:
-      <<: *cuda11_3_1
-    cuda_prefix_url: "rhel8"
+  # TODO: migrate to centos stream
```

Comment:
skipping centos base image release for 1.0.0a5 for now, due to CentOS 8 EOL. will discuss further with the team

Reply:
I will address ubi in a separate issues